build/configs/rtl8721csm/hello/defconfig

```diff
@@ -239,9 +239,9 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="128,"
-CONFIG_FLASH_PART_TYPE="smartfs,"
-CONFIG_FLASH_PART_NAME="user,"
+CONFIG_FLASH_PART_SIZE="16,8,512,128,"
+CONFIG_FLASH_PART_TYPE="bl,bl,kernel,smartfs,"
+CONFIG_FLASH_PART_NAME="bl,bl,kernel,userfs,"
```

Comment:
Should we name boot loader partitions "bl1" and "bl2" for clear separate identification in download scripts?
As is done in other defconfigs with 2 boot loader partitions?

Reply:
modified.